component/events.md

```diff
@@ -50,6 +50,205 @@ Components have the ability to bind to special [can.events.inserted inserted] an
         }
       }
 
+## Listening to viewModel events
+
+As in [can.Control] we can listening for events inside or outside the Component. We can use a Templated Event Handler for this. Unlike [can.Control] the value inside `{Name}` are looked up on `viewModel` first followed by `this.options` and then the `window` object.
+You can use the events for:
+* Tracking changes on [can.List] or [can.Map] like `add`, `remove`, `change`, (`replaced`)
+* Tracking changes on properties on the viewModel
+* Tracking changes on [can.Model]-Instance like `created`, `updated`, `destroyed`
+* Tracking promises on [can.Deferred] 
+
+The following example shows all the described possibilities:
```

Comment:
david meant https://github.com/canjs/canjs/issues/1740#issuecomment-149928218 this would be the best place.
since can.Controll isn't the future i think there would be a good place to show how you can listen to events thats fired in the viewModel.
this example indeet is long, but shows all possibilities. i think people can handle this

maybe we can add a reference to this documentation in can.List and can.Map (http://canjs.com/docs/can.Map.html#section_Listeningtochanges)